### ISSUE TITLE
Docker command in Medium article has an error?

### ISSUE THREAD

**Issue by simsong** [TamirMa/google-doorbel-telegram-sync#2]
Hi. This section in your article:
![image](https://github.com/TamirMa/google-nest-telegram-sync/assets/1594284/93ee1281-4b9e-4914-9f53-78c360d04e1c)

appears to have a typo? Here is what I get when I run it.
```
$ docker run - rm -it breph/ha-google-home_get-token
docker: invalid reference format.
See 'docker run --help'.
$
```
I am a docker newbie, so I'm not sure what you are suggesting here...

**Comment by TamirMa** [comment]
Hey, you are right, the command should be:
`docker run --rm -it breph/ha-google-home_get-token`

**Comment by simsong** [comment]
Great!  I don't think that a docker script is really needed to just run a python program, but if it helps some people, that's great.

Unfortunately, I wasn't able to get the token script to work. It's not clear to me why. Perhaps Google changed something.

**Comment by TamirMa** [comment]
You must run the docker to get a master token and use it in the Python script.
I don't think they changed something as I'm using it now..
If you share more details about your error, I would help you with that (:

**Comment by simsong** [comment]
All the docker container does is run a different python script!

The error I was getting was that the authorization was invalid. I tried it with an account protected by two-factor, with an application password on that account, and with another account that is not protected by two factor.  However, I was able to steal the OAuth token out of a browser, so I am now able to access the JPEGs. It would be nice to get this properly working with OAuth, however, which is well within my skillset if you would find that useful. 

I've been playing with https://nest-rtsp.jak.guru/ and haven't been able to get it to work — I keep getting a different kind of error. However, if I just download JPEGs, I can make my own rstp stream with ffmpeg.  Alternatively, I can build a video archiving system entirely based on JPEGs, although I won't get good full-motion video for events.  Looking at your code, it's not clear how you get full-motion video other than just downloading JPEGs and stringing them together.
